src/components/app/Home.js

```diff
@@ -236,16 +236,22 @@ class Home extends React.PureComponent<HomeProps, HomeState> {
   componentDidMount() {
     // Prevent dropping files on the document.
     // Help Flow infer the correct type signature for document.addEventListener.
+    // $FlowFixMe Error introduced by upgrading to v0.96.0.
     document.addEventListener(('drag': string), _preventDefault, false);
+    // $FlowFixMe Error introduced by upgrading to v0.96.0.
     document.addEventListener(('dragover': string), _preventDefault, false);
+    // $FlowFixMe Error introduced by upgrading to v0.96.0.
     document.addEventListener(('drop': string), _preventDefault, false);
     // Let the Gecko Profiler Add-on let the home-page know when it's been installed.
     homeInstance = this;
   }
 
   componentWillUnmount() {
+    // $FlowFixMe Error introduced by upgrading to v0.96.0.
     document.removeEventListener(('drag': string), _preventDefault, false);
```

Comment:
Maybe you can remove the `: string` in all these lines then? As well as the comment above. Unless you want to fix this before.

Reply:
I just hadn't finished the components yet, this is fixed.